include/CIC6105.h

```diff
@@ -0,0 +1,21 @@
+#ifndef CIC6105_H
+#define CIC6105_H
+
+#include "ultra64.h"
+
+// IPL3 puts values in those addresses as part of the anti-piracy checks
+#define CIC_ADDRESS_1 0x002FB1F4
+#define CIC_ADDRESS_2 0x002FE1C0
+
+
+void CIC6105_Nop80081820(void);
+void CIC6105_Nop80081828(void);
```

Comment:
```suggestion
void CIC6105_Noop1(void);
void CIC6105_Noop2(void);
```
Perhaps this is just me, but would prefer this instead.

Reply:
I prefer the opposite from a modding perspective: the address has more information than just a sequential number even if it's uglier.

... That said, the address is part of the build map, so I don't care that strongly about it and am fine with the change.

Reply:
While we have 1 version, I can see why `Nop80081820` would be nice, but as soon as we have multi-version support, then the name with the address wouldn't be great since it would be wrong for other versions, but `CIC6105_Noop1` would be fine for all versions. Still a while before we get to multi-version support though, so that's more long term. So `CIC6105_Noop1` is just more stable overall